internal/app/i18n/bundle.en.go

```diff
@@ -62,9 +62,10 @@ var en = map[Message]string{
 	ExtractingJdkFiles:                "Extracting JDK files to %s\n",
 	ExtractingJdkProgress:             "Extracting JDK... ",
 	ByeMessage:                        "Goodbye!",
-	UnableFetchPluginsError:           "Failed to fetch plugins from the generation server. Restart the app.",
+	UnableFetchPluginsError:           "Failed to fetch plugins from the server. Please restart the app.",
```

Comment:
I would ask a user to check the internet connection. @vnikolova, what do you think?

Reply:
If there are other possible reasons why the fetch failed, it could be nice to list them. Or if there's a link that could provide solution. I'd also remove the "please" (it's unnecessary)

Reply:
The most probable reason is a poor internet connection or a problem with our server. Both requires restarting the app.